homeassistant/components/fastdotcom/__init__.py

```diff
@@ -18,24 +16,18 @@ async def async_setup_entry(hass: HomeAssistant, entry: FastdotcomConfigEntry) -
     """Set up Fast.com from a config entry."""
     coordinator = FastdotcomDataUpdateCoordinator(hass, entry)
     entry.runtime_data = coordinator
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
```

Comment:
```suggestion
```
the integration uses `runtime_data`, so there is no need to store it in `hass.data` directly